frontend/js/src/fresh-releases/ReleaseCard.tsx

```diff
@@ -59,14 +63,11 @@ export default function ReleaseCard(props: ReleaseCardProps) {
   }
 
   React.useEffect(() => {
-    async function getCoverArt() {
-      const coverartURL = await getAlbumArtFromReleaseMBID(releaseMBID);
-      if (coverartURL) {
-        setCoverartSrc(coverartURL);
-      }
+    if (caaId && caaReleaseMbid) {
```

Comment:
Is there any point in having a fallback mechanism using `getAlbumArtFromReleaseMBID(releaseMBID)` as before? Is there a situation where that would be of any use?

Reply:
Done.